tests/test_edgeql_calls.py

```diff
@@ -520,16 +520,16 @@ class TestEdgeQLFuncCalls(tb.QueryTestCase):
             CREATE FUNCTION test::inner(
                 a: anytype
             ) -> int64
-                USING EdgeQL $$
```

Comment:
Wait, so this syntax is no longer supported?

Reply:
It is, I just cleaned up some tests to use more concise syntax.